docs/scripts_md/MRI.md

```diff
@@ -257,8 +257,10 @@ RETURNS: (int) CandID
 
 ### getPSC($patientName, $dbhr)
 
-Looks for the site alias in whatever field (usually patient\_name or
-patient\_id) is provided and return the MRI alias and CenterID.
+Looks for the site alias using the session table CenterID as a first
+resource, for the cases where it is created using the front-end,
+otherwise, find the site alias in whatever field (usually patient\_name 
+or patient\_id) is provided, and return the MRI alias and CenterID.
```

Comment:
idem, patient_name, patient_id and CenterID should be in code format

Reply:
done